SECURITY.md

```diff
@@ -0,0 +1,77 @@
+<p align="center">
+  <picture>
+    <source
+      width="256px"
+      media="(prefers-color-scheme: dark)"
+      srcset="assets/revanced-headline/revanced-headline-vertical-dark.svg"
+    >
+    <img 
+      width="256px"
+      src="assets/revanced-headline/revanced-headline-vertical-light.svg"
+    >
+  </picture>
+  <br>
+  <a href="https://revanced.app/">
+       <picture>
+           <source height="24px" media="(prefers-color-scheme: dark)" srcset="assets/revanced-logo/revanced-logo-round.svg" />
+           <img height="24px" src="assets/revanced-logo/revanced-logo-round.svg" />
+       </picture>
```

Comment:
These files are not present in the `compose-dev` branch.

Reply:
Done:tm: